foolbox/models/wrappers.py

```diff
@@ -21,3 +21,29 @@ def __call__(self, inputs: T) -> T:
         y = ep.where(x < self._threshold, min_, max_).astype(x.dtype)
         z = self._model(y)
         return restore_type(z)
+
+
+class ExpectationOverTransformationWrapper(Model):
```

Comment:
Sorry for not mentioning this earlier, but can you please add docstrings to the class & module like its done for the other classes in the project?